webview/src/setup/components/dvc/Dvc.tsx

```diff
@@ -57,10 +59,16 @@ export const Dvc: React.FC = () => {
   if (!projectInitialized) {
     return <ProjectUninitialized>{children}</ProjectUninitialized>
   }
+
   return (
     <EmptyState isFullScreen={false}>
       <h1>Setup Complete</h1>
       {children}
+      <p>
+        New to the extension? Go through the walkthrough to familiarize yourself
+        with the different features.
+      </p>
+      <Button onClick={showWalkthrough} text="Show Walkthrough" />
```

Comment:
[Q] Should this be shown the entire time?

Reply:
or does it distract from the real purpose of the section?

Reply:
> [Q] Should this be shown the entire time?

Would be nice to only show it for new users but I'm not seeing a way to detect that. We could show the content if the user went through the setup (like we do with autoclosing) but could that possibly lead to unexpected behavior? For example, if a user goes through the setup but comes back after closing the webview, they will no longer see the text/button 🤔 

> or does it distract from the real purpose of the section?

In my opinion, I don't think it's distracting. We expect new users to take a look at the walkthrough after setting up DVC so it makes sense to me to have a "Show Walkthrough" button there. 

But ultimately no strong preference, I'll wait a bit before merging and we can always revert/update in a followup!